server/routes/aws/securityhub_routes.py

```
@@ -0,0 +1,161 @@
+def get_findings(user_id):
+    """
+    Fetch AWS Security Hub findings for the authenticated user's organization.
+    Supports a limit parameter bounded between 1 and 200.
+    """
+    org_id = get_org_id_from_request()
+    limit = request.args.get('limit', 50, type=int)
+    limit = max(1, min(limit, 200))
+    
+    try:
+        with db_pool.get_admin_connection() as conn:
+            with conn.cursor(cursor_factory=RealDictCursor) as cursor:
```

Ruff (0.15.12)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)